tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import zipfile
+from pathlib import Path
+
+
+@pytest.fixture
+def create_repo(tmpdir, autouse=True):
+    with zipfile.ZipFile(str(Path(__file__).parent / 'repo_data.zip'), 'r') as z:
+        z.extractall(str(Path(tmpdir)))
```

Comment:
I think it would be more transparent to
- name the fixture `tmprepo`
- return `Path(tmpdir)`

Then, tests using the fixture don't need to also require the `tmpdir` fixture and it will be more transparent what's going on.

I also don't understand what `autouse=True` is doing here.

Reply:
I still think, providing the test data as plain files and not in a zip archive would be simpler and more transparent.

Reply:
ok - it seems avoiding a zip file is much better

Reply:
In that case, the fixture would just `shutil.copytree` the repository to `tmpdir`.

Reply:
tests for py3.5 passes - should we increase the pytest version to run all others?